test/e2e/cypress/integration/single-backend/users.spec.js

```diff
@@ -284,11 +284,36 @@ describe('Users', function() {
 
     cy.get(
       '[data-cy="UserManagement-pagination"] .page-link[aria-posinset="2"]'
-    ).click({force: true})
+    ).click({ force: true })
     cy.get('[data-cy=UserItem]').should('have.length', 4)
     cy.url().should('contain', 'from=10')
   })
 
+  it('Should be able to lists the users with a wrong from url parameter', () => {
+    if (Cypress.env('BACKEND_VERSION') === 2) {
```

Comment:
Instead of this, you should actually use `cy.skipOnBackendVersion(1)` that @Njuelle developed